kong/templates/nginx_kong.lua

```diff
@@ -32,6 +32,7 @@ lua_shared_dict kong                5m;
 lua_shared_dict kong_cache          ${{MEM_CACHE_SIZE}};
 lua_shared_dict kong_process_events 5m;
 lua_shared_dict kong_cluster_events 5m;
+lua_shared_dict kong_healthchecks   5m;
```

Comment:
just doing due diligence, have we investigated that this is a large enough allocation? and that the healthcheck library handles and alerts on OOM/eviction conditions?

Reply:
Large enough is always relative, but yes, the library does alert on shm allocation failures. This PR logs whatever comes from the healthcheck library.